conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+* Location: University of Birmingham
+* Date: 17th - 19th September 2019
+
+### Workshop Length
+
+180 mins (to allow time for Kubernetes cluster deployment and an introduction to reproducible research and BinderHub).
+
+### Title
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
+It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user.
+A unique URL to this browser is generated making the software easily sharable with collaborators.
+BinderHubs can be built anywhere in the world and are Cloud-neutral.
+Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
+
+By harnessing the power of Cloud computing and building a BinderHub with expanded capabilities, scientific analyses can become more accessible and transparent to the wider community.
+BinderHubs can showcase the reproducibility of software as well as being a valuable teaching or demonstration tool.
+
+In this workshop, attendees will learn how to deploy a BinderHub on Microsoft Azure and connect with others who have or are interested in doing so.
+Attendees should be familiar with Azure Cloud computing and the command line.
+
+### Audience
+
+_Would your target audience be required to have any prerequisite skills/background knowledge e.g. knowledge of a particular language?_
+
+Attendees would preferably be familiar with Microsoft Azure and the command line.
+
+### Outcomes
+
+_How will your attendees benefit from your session? What do you expect them to gain/learn?_
+
+_(600 characters max, current count: 524 [please update if you edit])_
+
+Attendees to this workshop will learn how to deploy a BinderHub on Microsoft Azure Cloud platform.
+They will learn the multiple applications of a BinderHub as a teaching or demonstration tool; the technology powering a BinderHub; and why reproducibility is important in research.
+Attendees will be provided with enough knowledge to make an informed decision on whether building their own BinderHub is advantageous to their goals and connect with other RSEs/researchers who have build a BinderHub or are interested in doing so.
+
+### Accessibility
+
+_How will you ensure that your content will be accessible to the diverse audience of the conference?_
```

Comment:
I think this question is asking about how you're going to welcome people and teach them something that they can follow along with.

I'd write here something about how the BinderHub tutorial doesn't require much background knowledge or expertise. I'd also link to the binderhub workshop and talk about the "traffic light" post its to make sure you don't leave anyone behind.

I'd be looking for you to be clear about who is helping you and the ratio of support you can provide.

You can also note here that you've built in time for questions and discussion.

Reply:
"I will make clear at the beginning of the workshop that all participants are required to comply with the RSE19 code of conduct at all time. I will make sure that I know the reporting and enforcement details so that I can effectively support any member of the community in the event that they feel unsafe during my session."

Reply:
Done in [`4c56572`](https://github.com/alan-turing-institute/the-turing-way/pull/425/commits/4c56572370172ae0e9810a2adedfb88bce7e6deb) and [`870993b`](https://github.com/alan-turing-institute/the-turing-way/pull/425/commits/870993ba09958add7f906e45e1c29d4f48dfb53f)